### PR TITLE
Include namespace in api_dead_package

### PR DESCRIPTION
**Pull request by apevec**
Fixes https://github.com/fedora-infra/pkgdb2/issues/319

### CODE REVIEW

File: pkgdb2/api/extras.py [hunk 1]
```diff
@@ -625,7 +625,8 @@ def api_koschei():
 
 
 @API.route('/dead/package/<pkg_name>/<clt_name>')
-def api_dead_package(pkg_name, clt_name):
+@API.route('/dead/package/<namespace>/<pkg_name>/<clt_name>')
```
Comment (pypingou) [inline pkgdb2/api/extras.py]: This won't be backward compatible, we need either to support the old route or to add a second one

Comment (apevec) [inline pkgdb2/api/extras.py]: This is exactly what I did here, added a second route.

Comment (pypingou) [inline pkgdb2/api/extras.py]: Oh, stupid me, sorry about that

**Comment by pypingou** [comment]
Tested locally as well, works great, thanks a lot for catching and fixing this :)

**Pull request merged**